build.gradle

```diff
@@ -2,7 +2,9 @@ buildscript {
     ext.kotlinVersion = '1.3.50'
     ext.appCompatVersion = '1.1.0'
     ext.recyclerViewVersion = '1.0.0'
+    ext.constraintLayoutVersion = '1.1.3'
     ext.lifecycleVersion = '2.1.0'
+    ext.coreVersion = '1.1.0'
```

Comment:
core is a generic name here. this could probably be `androidXCoreKtx` (lots of libraries have "core" in the module names"